### PR TITLE
[ocaml] print more verbose info on xml loading errors

### PR DESCRIPTION
**Pull request by flixr**
Try to use our own `ExtXml.parse_file` instead of plain `Xml.parse_file` in most cases to get a better error report if the xml file is invalid.
e.g.
```
/home/flixr/code/paparazzi/conf/airframes/examples/bebop.xml: End of tag expected : 'load' line 43 characters 2-11
make: *** [ap.ac_h] Error 1
```
instead of only
```
Fatal error: exception Xml.Error(_)
make: * [ap.ac_h] Error 2
```

### CODE REVIEW

**Comment by OpenUAS** [comment]
Very welcomed, I know a 100(student) reasons why ;) 
Ideas:
For users encountering issues, the "make: *** [ap.ac_h] Error 1" likely is not all that informative an IMHO can be chopped of the message. 

And we could add if XMLerror encountered start with something descriptive to make sure it is a user error that can be fixed.

"There is an Error in your XYZ.shortname.XML you can fix. Your problem to fix is details are:"

**Comment by flixr** [comment]
I don't quite follow... is the message printed with this update not descriptive enough?

File: sw/ground_segment/cockpit/sectors.ml [hunk 1]
```diff
@@ -32,7 +32,7 @@ let load = fun geomap () ->
       None -> ()
     | Some f ->
       try
-        let xml = Xml.parse_file f in
+        let xml = ExtXml.parse_file f in
```
Comment (gautierhattenberger) [inline sw/ground_segment/cockpit/sectors.ml]: not sure this one should be changed, it breaks the `try ... catch` and the popup message

File: sw/ground_segment/cockpit/sectors.ml [hunk 2]
```diff
@@ -44,7 +44,7 @@ let load_kml = fun geomap () ->
...
```
Comment (gautierhattenberger) [inline sw/ground_segment/cockpit/sectors.ml]: same here

File: sw/tools/generators/gen_autopilot.ml [hunk 3]
```diff
@@ -423,7 +423,7 @@ let write_settings = fun xml_file out_set ap ->
   *)
 let gen_autopilot main_freq xml_file h_dir out_set =
   try
-    let ap_xml = Xml.parse_file xml_file in
+    let ap_xml = ExtXml.parse_file xml_file in
```
Comment (gautierhattenberger) [inline sw/tools/generators/gen_autopilot.ml]: DTD errors were already caught here

File: sw/tools/generators/gen_mtk.ml [hunk 4]
```diff
@@ -125,7 +125,7 @@ let _ =
   end;
   let xml_file = Sys.argv.(1) in
   try
-    let xml = Xml.parse_file xml_file in
+    let xml = ExtXml.parse_file xml_file in
```
Comment (gautierhattenberger) [inline sw/tools/generators/gen_mtk.ml]: DTD errors already caught

File: sw/tools/generators/gen_ubx.ml [hunk 5]
```diff
@@ -155,7 +155,7 @@ let _ =
   end;
   let xml_file = Sys.argv.(1) in
   try
-    let xml = Xml.parse_file xml_file in
+    let xml = ExtXml.parse_file xml_file in
```
Comment (gautierhattenberger) [inline sw/tools/generators/gen_ubx.ml]: DTD errors already caught here

File: sw/tools/generators/gen_xsens.ml [hunk 6]
```diff
@@ -224,7 +224,7 @@ let _ =
   end;
   let xml_file = Sys.argv.(1) in
   try
-    let xml = Xml.parse_file xml_file in
+    let xml = ExtXml.parse_file xml_file in
```
Comment (gautierhattenberger) [inline sw/tools/generators/gen_xsens.ml]: DTD errors caught here

File: sw/tools/generators/gen_autopilot.ml (hunk 3 above)
Comment (flixr) [inline sw/tools/generators/gen_autopilot.ml]: does it make sense to keep duplicating the error handling here instead of using ExtXml.parse_file?

**Comment by flixr** [comment]
@gautierhattenberger thanks for the corrections, updated...

**Pull request merged**